packages/graphql-yoga/__tests__/error-masking.spec.ts

```diff
@@ -859,4 +860,49 @@ describe('error masking', () => {
 }
 `);
   });
+
+  it.only('should mask experimental coordinate error attribute on production env', async () => {
+    const yoga = createYoga({
+      logging: false,
+      maskedErrors: {
+        isDev: true,
+      },
+      plugins: [useErrorCoordinate()],
+      schema: createSchema({
+        typeDefs: /* GraphQL */ `
+          type Query {
+            a: String!
+          }
+        `,
+        resolvers: {
+          Query: {
+            a: () => {
+              throw createGraphQLError('Test Error', { coordinate: 'Query.a' });
```

Comment:
```suggestion
              throw createGraphQLError('Test Error');
```
Shouldn't the coordinate get added automatically?

Reply:
Oh you are right, it's a leftover from my debugging with the serialization issue I had yesterday... which ended up just being the `toJSON` method :-P